File: Python/2901-2950/2942. Find Words Containing Character.py
# https://leetcode.com/problems/find-words-containing-character/

class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        result = []

        for index in range(0, len(words)):
            if x in words[index]:
                result.append(index);

        return result
        

File: Java/2901-2950/2942. Find Words Containing Character.java
// https://leetcode.com/problems/find-words-containing-character/

class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> result = new ArrayList<>();

        for(int index=0; index<words.length; index++){
            if(words[index].contains(x+""))
                result.add(index);
        }

        return result;
    }
}
